analyzers/src/SonarAnalyzer.VisualBasic/Rules/MethodParameterUnused.cs

```diff
@@ -66,92 +63,71 @@ protected override void Initialize(SonarAnalysisContext context)
+                    // Bail-out if this is not a method we want to report on (only based on symbols checks)
                     var methodSymbol = c.SemanticModel.GetDeclaredSymbol(methodBlock);
-                    if (methodSymbol == null ||
-                        methodSymbol.IsAbstract ||
-                        methodSymbol.IsMainMethod() ||
-                        methodSymbol.IsEventHandler() ||
-                        methodSymbol.GetEffectiveAccessibility() != Accessibility.Private)
+                    if (methodSymbol == null
+                        || methodSymbol.IsAbstract
+                        || methodSymbol.IsMainMethod()
+                        || methodSymbol.IsEventHandler()
+                        || methodSymbol.GetEffectiveAccessibility() != Accessibility.Private)
                     {
                         return;
                     }
 
                     foreach (var parameter in unusedParameters)
                     {
-                        c.ReportDiagnosticWhenActive(Diagnostic.Create(rule, parameter.GetLocation(),
-                            parameter.Identifier.Identifier.ValueText));
+                        c.ReportDiagnosticWhenActive(Diagnostic.Create(Rule, parameter.GetLocation(), parameter.Identifier.Identifier.ValueText));
                     }
                 },
                 SyntaxKind.SubBlock,
                 SyntaxKind.FunctionBlock);
-        }
 
         private static bool HasAnyParameter(MethodBlockBaseSyntax method) =>
             method.BlockStatement.ParameterList != null
-            && method.BlockStatement.ParameterList.Parameters.Count > 0;
+            && method.BlockStatement.ParameterList.Parameters.Any();
 
         private static bool IsEmptyMethod(MethodBlockBaseSyntax method) =>
             method.Statements.Count == 0;
 
         private static bool IsVirtualOrOverride(MethodBlockBaseSyntax method) =>
-             method.BlockStatement.Modifiers.Any(m =>
-                m.IsKind(SyntaxKind.OverridesKeyword) ||
-                m.IsKind(SyntaxKind.OverridableKeyword));
+             method.BlockStatement.Modifiers.Any(x => x.IsAnyKind(SyntaxKind.OverridesKeyword, SyntaxKind.OverridableKeyword));
 
         private static bool IsInterfaceImplementation(MethodBlockBaseSyntax method) =>
             (method.BlockStatement as MethodStatementSyntax)?.ImplementsClause != null;
 
         private static bool HasAnyAttribute(MethodBlockBaseSyntax method) =>
-            method.BlockStatement.AttributeLists.Count > 0;
+            method.BlockStatement.AttributeLists.Any();
 
         private static bool OnlyThrowsNotImplementedException(MethodBlockBaseSyntax method, SemanticModel semanticModel) =>
             method.Statements.Count == 1
             && method.Statements
                 .OfType<ThrowStatementSyntax>()
-                .Select(tss => tss.Expression)
+                .Select(x => x.Expression)
```

Comment:
yes here `x` is much better than `tss`, `oces` or `s` :)

Reply:
I found `oces` especially hard to understand. Even when it became obvious when you finally find it :)